### ISSUE TITLE
Error syncing market data after update to 1.6.1

### ISSUE THREAD

**Issue by johansigfrids** [AugurProject/augur-app#384]
After the update to 1.6.1 I am getting the following error when attempting to sync market data:

```
insert into `token_supply` (`supply`, `token`) select '0' as `supply`, '0xf4b0d6d4926d69a40bff3251ae3cbd8cc092b6b4' as `token` union all select '0' as `supply`, '0x82f0a32ab18a3be686e059316eb5139c422775fe' as `token` - SQLITE_CONSTRAINT: UNIQUE constraint failed: token_supply.token
```

**Comment by nuevoalex** [comment]
Made an issue for this one for us to track internally: [issue](https://app.clubhouse.io/augur/story/17134/token-supply-row-not-removed-in-market-created-log-removal)

**Comment by larspensjo** [comment]
I have the same problem.

**Comment by veox** [comment]
I think this has been fixed in [v1.6.2](https://github.com/AugurProject/augur-app/releases/tag/v1.6.2)?..

**Comment by larspensjo** [comment]
Doesn't work with 1.6.2 for me, I get the same error. Doing a "reset database" doesn't help.

**Comment by veox** [comment]
@larspensjo Do you mean that "reset database" doesn't reset the database, or that the issue re-surfaces after re-syncing?

(I've encountered the former a few times, where clicking "reset database" doesn't really do anything.)

**Comment by larspensjo** [comment]
I don't know if the database was reset. I just tried to click that option, and see if anything improved. Maybe it is irrelevant to the issue.

I would really like to know how I can make a new clean install, and start everything fresh again. If I uninstall the Augur app, and re-install an older version, there seems to be some saved data somewhere that prevents it from working.

**Comment by larspensjo** [comment]
Thanks, renaming the "augur" folder forced a reset, and now 1.6.2 now works for me.

**Comment by veox** [comment]
OK, looks like there's an unrelated issue that could use being reported. ;)

**Comment by adrake33** [comment]
Closing this issue, as the DB reset bug should be fixed when version 1.7.0 is released.
